app/constants.py

```diff
@@ -0,0 +1,13 @@
+from web3 import Web3
+
+# Eternally constant constants
+
+ZERO_ADDR = "0x0000000000000000000000000000000000000000"
+ZERO_ADDR_BYTES = Web3.toBytes(hexstr=ZERO_ADDR)
```

Comment:
Can we import a smaller function for this operation?

Reply:
Valid question.